cmd.go

```diff
@@ -14,7 +14,7 @@ type DirLister interface {
 }
 
 func (c CmdUnix) ListContent(path string) ([]string, error) {
-	cmd := exec.Command("ls", "-a", path)
+	cmd := exec.Command("ls", path)
```

Comment:
Could you please keep the `"-a"` argument? It handles hidden files.

Reply:
Sure!